packages/happo-target-firefox/src/findBackgroundImageUrls.js

```diff
@@ -5,7 +5,9 @@ export default function findBackgroundImageUrls(string) {
   let match;
   // eslint-disable-next-line no-cond-assign
   while (match = URL_PATTERN.exec(string)) {
-    result.push(match[1]);
+    // We need to unescape the simple quote.
```

Comment:
This comment could be expanded a little to better explain why it's here. How about

> Inlined images of the form `data:image` can contain embedded svg. To avoid failing to load this image, we need to unescape escaped single quotes.